src/charm.py

```diff
@@ -324,6 +329,12 @@ def _reconcile_ha(self, ha_information: HAInformation) -> None:
         self.hacluster.bind_resources()
         peer_relation.data[self.unit].update({"vip": str(ha_information.vip)})
 
+    @validate_config_and_tls(defer=False)
+    def _on_haproxy_route_data_available(self, _: HAServiceReadyEvent) -> None:
+        """Handle the ha-ready event."""
+        data = self.haproxy_route_provider.get_data(self.haproxy_route_provider.relations)
+        logger.info("%s", data)
```

Comment:
debug?

Reply:
updated it to debug and also added a comment to explain why it's there